src/main/java/com/pscheduler/server/core/DatabaseLoader.java

```diff
@@ -1,9 +1,11 @@
 package com.pscheduler.server.core;
 
 import com.pscheduler.server.model.Course;
+import com.pscheduler.server.model.CourseGPA;
 import com.pscheduler.server.model.Meeting;
 import com.pscheduler.server.repository.CourseRepository;
 import com.pscheduler.server.repository.MeetingRepository;
+import com.pscheduler.server.repository.CourseGPARepository;

```

Comment:
very nitpicky, this import isn't in alphabetical order. I usually let intellij do the import for me using alt+enter

Reply:
Fixed.